src/Migration/MmConfigMigration.php

```diff
@@ -0,0 +1,187 @@
+                    $config['polyfillEnable'] = (int) $module['dk_mmenuPolyfillEnable'];
+                    $config['keyboardNavigation'] = (int) $module['dk_mmenuKeyboardNavigation'];
+                    $config['keyboardNavigationEnhance'] = (int) $module['dk_mmenuKeyboardNavigationEnhance'];
+
+                    $stmt = $this->connection->prepare(
+                        'INSERT INTO tl_dk_mmenu_config (
+                                `title`,
+                                `position`,
+                                `zposition`,
+                                `slidingSubmenus`,
+                                `theme`,
+                                `moveBackground`,
+                                `pageDim`,
+                                `fullscreen`,
+                                `countersAdd`,
+                                `columnsAdd`,
+                                `searchfieldAdd`,
+                                `iconPanels`,
+                                `menuEffects`,
+                                `panelEffects`,
+                                `listEffects`,
+                                `shadows`,
+                                `onClickClose`,
+                                `pageSelector`,
+                                `dragOpenEnable`,
+                                `dragOpenMaxStartPos`,
+                                `dragOpenThreshold`,
+                                `polyfillEnable`,
+                                `keyboardNavigation`,
+                                `keyboardNavigationEnhance`
+                            ) VALUES (
+                                :title,
+                                :position,
+                                :zposition,
+                                :slidingSubmenus,
+                                :theme,
+                                :moveBackground,
+                                :pageDim,
+                                :fullscreen,
+                                :countersAdd,
+                                :columnsAdd,
+                                :searchfieldAdd,
+                                :iconPanels,
+                                :menuEffects,
+                                :panelEffects,
+                                :listEffects,
+                                :shadows,
+                                :onClickClose,
+                                :pageSelector,
+                                :dragOpenEnable,
+                                :dragOpenMaxStartPos,
+                                :dragOpenThreshold,
+                                :polyfillEnable,
+                                :keyboardNavigation,
+                                :keyboardNavigationEnhance
+                            )'
+                    );
+                    $stmt->executeQuery($config);
+                    $configId = (int) $this->connection->lastInsertId();
```

Comment:
Is there a bette way to get the last config id of the insert?

Reply:
I don't think there is another way.